build_tools/docker/README.md

```diff
@@ -93,21 +93,4 @@ python3 build_tools/docker/manage_images.py --image all
 4. Commit the changes and send a PR for review. The CI will use the updated
    digest references to test the new images.
 
-5. Merge your PR after is approved and all CI tests pass. **Please remember to
-   complete the step below**.
-
-### Part 3. Updating the `:prod` tag
-
-Kokoro builds preload images tagged with `prod` on VM creation, so after
-changing the images used, you should also update the images tagged as `prod`
-in GCR. This also makes development significantly easier for others who need to
-modify the `docker` images.
-
-6. We use `build_tools/docker/prod_digests.txt` as a source of truth for which
-   versions of the images on GCR should have the `:prod` tag. The following
-   command will ensure that you are at upstream HEAD on the `main` branch before
-   it updates the tags.
-
-    ```shell
-    python3 build_tools/docker/manage_prod.py
-    ```
```

Comment:
This looks unrelated to your change, and I'm not sure it's quite right to just remove these docs. We don't use Kokoro anymore, but the manage_prod.py script could still be useful?

Reply:
The `prod_digest.txt` is updated in the previous step, as well as all the .github/action yaml instances, so once the PR is merged, we don't need to update the tag anymore.

You are right about the two commits not directly related. I plan to merge it individually instead of squashing. Please let me know if you think I should create another PR for the markdown update.

Reply:
We need @GMNGeoffrey to chime in on this. I will move this commit to another PR and we can iterate over there.

Reply:
We currently just use the prod_digests.txt file as the source of truth on what is the prod image. The tags were needed so Kokoro could pre-fetch them, but they aren't used for anything anymore and nothing reliably updates them (I don't think people were consistently following these instructions), so I think it's probably best to just drop this and the manage_prod.py script

Reply:
Got it. will update the doc and remove the script in a different PR. Thanks for the input!